test/old/testdir/test_normal.vim

```diff
@@ -3905,7 +3905,7 @@ func Test_normal_gj_on_extra_wide_char()
   put =text
   call cursor(2,1)
   norm! gj
-  call assert_equal([0,2,25,0], getpos('.'))
+  call assert_equal([0,2,29,0], getpos('.'))
```

Comment:
This test is for unprintable character. When ZWJ is no longer unprintable some other unprintable character between 0x1000 and 0xFFFF should be used.